docs/user/spark/spark_jts.rst

```diff
@@ -125,6 +125,23 @@ also achievable with the following code:
     import spark.implicits. _
     chicagoDF.where(st_contains(st_makeBBOX(0.0, 0.0, 90.0, 90.0), $"geom"))
 
+GeoTools User-defined Functions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
[format] the header here has to match the length of the title

Reply:
Okay, update done.